src/components/UI/atoms/Divider/Divider.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { borderColors } from '<variables>/colorPalette';
+import spacing from '<variables>/spacing';
+
+/**
+ * @description - Divider Component
+ *
+ * @prop {string} border - border
+ * @prop {string} color - color
+ * @prop {string} width - width
+ * @prop {string} display - display
+
+ *
+ * @return {component} Divider
+ */
+const Divider = ({
+  width,
+  color,
+  display,
+  children,
+}) => (
+  <Divider.Container
+    width={width}
+    display={display}
```

Comment:
Remove this as dividers are mostly going to be block

Reply:
I used span for the divider and span are inline by default so I need to set the display to block

Reply:
if this is the case then i suggest you use div then

Reply:
Alright